docs/faq/code-analysis/does-codacy-check-for-dependencies.md

```diff
@@ -1,5 +1,5 @@
 # Does Codacy check for dependencies?
 
-Codacy doesn't check for dependencies. Codacy performs static code analysis on the source code and checks for code style and security issues, duplication, complexity, and coverage.
+Yes, Codacy scans the manifest files of your repositories and displays any vulnerable dependencies as issues in the [Security monitor](../../repositories/security-monitor.md).
```

Comment:
do we want to promote the Security Monitor if we're thinking about removing it? Could we say the Issues page instead? 🤔

Reply:
indeed the Security Monitor (until we migrate it) is the only place where you can filter by all vulnerable dependencies (InsecureModuleLibraries category)

Reply:
Agreed. Let's simply call them "Codacy issues".

```suggestion
Yes, Codacy scans the manifest files of your repositories and displays any vulnerable dependencies as Codacy issues.
```